src/enki.py

```diff
@@ -28,42 +23,52 @@
     if not os.path.exists(item):
         print(f"\nENKI ERROR: '{item}' doesn't exist in your repository.")
         args.path.remove(item)
-        continue
+        sys.exit(2)
 
 user_input = args.path
 
 
-if args.command == 'generate':
+def expand_file_paths(item):
+    """Expand filepaths."""
+    expanded_files = []
+
+    for dirpath, dirnames, filenames in os.walk(str(item) + '/'):
+        for name in filenames:
+            if name.startswith('_') or not name.endswith('adoc') or name == 'master.adoc' or name == 'README.adoc':
```

Comment:
Because of the missing dot in `name.endswith('adoc')`, the script will incorrectly process all files that end with this string and not just those with the `.adoc` file extension:

```
$ enki validate .

VALIDATION ERROR: additional resources tag without the Additional resources header was found in the following files:
        boo.radoc
        kroupadoc
```
This is inconsistent with the behavior for explicitly supplied files which use `name.endswith('.adoc')`